public-api/src/test/java/org/pmiops/workbench/cdr/TestSqlDialect.java

```diff
@@ -0,0 +1,18 @@
+package org.pmiops.workbench.cdr;
+
+import org.hibernate.dialect.function.SQLFunctionTemplate;
+import org.hibernate.type.StandardBasicTypes;
+
+public class TestSqlDialect extends org.hibernate.dialect.MySQLDialect {
```

Comment:
Instead of copying this, we should probably put it in common-api/src/test, and change the build to include it in both api and public-api tests.

Reply:
Yeah. That would help.
It complained when I used the MySqlDialect which is already present in common-api. 
So made this one.